src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -40,6 +40,8 @@
   public static final String TOPIC_KEY_IGNORE_CONFIG = "topic.key.ignore";
   public static final String SCHEMA_IGNORE_CONFIG = "schema.ignore";
   public static final String TOPIC_SCHEMA_IGNORE_CONFIG = "topic.schema.ignore";
+  public static final String CONNECTION_TIMEOUT_CONFIG = "conn.timeout.ms";
```

Comment:
Recommend changing the variable name to `CONNECTION_TIMEOUT_MS_CONFIG` to maintain consistent naming pattern.

Reply:
good point. changed. thanks!